plugins/UsersManager/API.php

```diff
@@ -879,6 +879,10 @@ private function enrichUser($user)
         unset($user['password']);
         unset($user['ts_password_modified']);
 
+        if ($lastSeen = LastSeenTimeLogger::getLastSeenTimeForUser($user['login'])) {
+            $user['last_seen'] = date("Y-m-d H:i:s", $lastSeen);
```

Comment:
Guess we could use this here:
```suggestion
            $user['last_seen'] = Date::getDatetimeFromTimestamp($lastSeen);
```